src/windows.rs

```diff
@@ -107,3 +110,172 @@ impl PutAddress for ProcessHandle {
         }
     }
 }
+
+/// Use `CreateToolhelp32Snapshot` to get and filter list of loaded modules (called DLLs in Windows)
+/// of this process, returning the base address of it.
+#[allow(clippy::clippy::cast_possible_truncation)] // for size_of as u32
+impl ModuleInfo for Pid {
+    fn get_module_base(&self, name: &str) -> std::io::Result<usize> {
+        // taken from https://stackoverflow.com/questions/41552466/how-do-i-get-the-physical-baseaddress-of-an-dll-used-in-a-process
+        let mut module_entry = tlhelp32::MODULEENTRY32 {
+            dwSize: 0,
+            th32ModuleID: 0,
+            th32ProcessID: 0,
+            GlblcntUsage: 0,
+            ProccntUsage: 0,
+            modBaseAddr: std::ptr::null_mut(), // yikes
```

Comment:
why "yikes"?

Reply:
Null pointers. No way around it in this case, but always yikes and you won't convince me otherwise.